chpl/chpl-resources/src/main/resources/errors.properties

```
@@ -1019,3 +1019,17 @@ subscription.objectTypeIdRequired=An object type for the subscription is require
 subscription.objectTypeIdInvalid=The provided object type ID '%s' is not valid.
 subscription.objectIdRequired=A subscribed object ID is required.
 subscription.objectIdInvalid=The subscribed object ID '%s' is not valid.
+
+criteriaAttribute.edit.emptyValue=Value is required.
+criteriaAttribute.edit.emptyRegulatoryTextCitation=Regulatory Text Citation is required.
+criteriaAttribute.edit.noCriteria=A %s requires at least one Criterion associated with it.
+criteriaAttribute.edit.duplicate=Regulatory Text Citation and Value matches an existing %s.
+criteriaAttribute.edit.deletedCriteria.listingsExist=This %s and Criteria %s is attested to by %s Listing%s
+criteriaAttribute.edit.emptyStartDay=Start Day is required.
+criteriaAttribute.edit.emptyEndDay=End Day is required.
+criteriaAttribute.edit.emptyRequiredDay=Required Day is required.
```

I think we generally use the word "Date" as our display value for these dates, even though we're using LocalDates with `Day` variable names
```suggestion
criteriaAttribute.edit.emptyStartDay=Start Date is required.
criteriaAttribute.edit.emptyEndDay=End Date is required.
criteriaAttribute.edit.emptyRequiredDay=Required Date is required.
```